.gitignore

```diff
@@ -2,6 +2,7 @@
 node_modules
 /build
 /.svelte-kit
+/.github
```

Comment:
Why did you add this folder to gitignore? And it's even already committed

Reply:
> Why did you add this folder to gitignore? And it's even already committed

yes, this does not make sense =)